### PR TITLE
change structure slightly and fix port typo

### PR DESCRIPTION
**Pull request by Topener**

### CODE REVIEW

File: .gitignore [hunk 1]
```diff
@@ -18,3 +18,4 @@
 npm-debug.log*
 yarn-debug.log*
 yarn-error.log*
+.vscode/settings.json
```
Comment (HendrikThePendric) [inline .gitignore:21]: I think you might as well gitignore the entire folder.

**Review by HendrikThePendric — APPROVED** [review]
LGTM. One minor suggestion.

File: docs/quickstart/quickstart-web.md [hunk 2]
```diff
@@ -22,7 +26,7 @@ cd my-app && yarn start
 
 ### Connecting your Web app to DHIS2
 
-1. From the browser, navigate to your DHIS2 instance url (if you're running DHIS2 locally via container the address is [http://localhost:3000](http://localhost:3000)). You will see the following page:
+1. From the browser, navigate to your DHIS2 instance URL (if you're running DHIS2 locally via container the address is [http://localhost:3000](http://localhost:8080)). You will see the following page:
```
Comment (tomzemp) [inline docs/quickstart/quickstart-web.md:29]: Should this be `[http://localhost:8080](http://localhost:8080)`? 

I'm confused by this line actually...shouldn't the web app open in http://localhost:3000 regardless of how you're running DHIS2? Maybe, it's just how I'm reading it, but it seems like we'd want to say the web app opens in http://localhost:3000 and the DHIS2 server will be at http://localhost:8080 if you've followed the instructions for running DHIS2 locally via container?

Comment (Topener) [reply docs/quickstart/quickstart-web.md]: yep you're right! Thanks!

**Pull request merged**